core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -212,11 +212,11 @@ oppia.factory('oppiaPlayerService', [
           loadedExploration.then(function(data) {
             exploration = ExplorationObjectFactory.createFromBackendDict(
               data.exploration);
-            version = exploration.version;
+            version = data.version;
```

Comment:
This is because reader.py returns version outside of the exploration object. I did not notice that on my first pass.